tests/unit/customizations/test_commands.py

```diff
@@ -144,45 +144,101 @@ def create_help_command(self):
-        rendered = help_command.doc.getvalue().decode('utf-8')
-        # The links aren't generated in the "man" mode.
-        self.assertIn(
-            "See 'aws help' for descriptions of global parameters", rendered
-        )
+        operation_handler.doc_synopsis_end(help_command=help_command)
+        rendered = help_command.doc.getvalue()
+        rendered = rendered.decode('utf-8')
+        for arg in help_command.arg_table:
+            arg_name = help_command.arg_table.get(arg).cli_name
+            self.assertNotIn(arg_name, rendered)
 
-    def test_includes_global_args_ref_in_html_options(self):
+    def test_includes_global_options_man(self):
+        help_command = self.create_help_command()
+        help_command.arg_table = self.create_arg_table()
+        operation_handler = BasicDocHandler(help_command)
+        operation_handler.doc_global_option(help_command=help_command)
+        rendered = help_command.doc.getvalue()
+        rendered = rendered.decode('utf-8')
+        for arg in help_command.arg_table:
+            arg_name = help_command.arg_table.get(arg).cli_name
+            type_name = help_command.arg_table.get(arg).cli_type_name
+            target = f'``{arg_name}`` ({type_name})'
+            self.assertIn(target, rendered)
+
+    def test_not_includes_global_options_man(self):
+        help_command = self.create_help_command()
+        operation_handler = BasicDocHandler(help_command)
+        operation_handler.doc_global_option(help_command=help_command)
+        rendered = help_command.doc.getvalue()
+        rendered = rendered.decode('utf-8')
+        for arg in help_command.arg_table:
+            arg_name = help_command.arg_table.get(arg).cli_name
+            type_name = help_command.arg_table.get(arg).cli_type_name
+            target = f'``{arg_name}`` ({type_name})'
+            self.assertNotIn(target, rendered)
+
+    def test_includes_global_options_html(self):
         help_command = self.create_help_command()
         help_command.doc.target = 'html'
+        help_command.arg_table = self.create_arg_table()
         operation_handler = BasicDocHandler(help_command)
-        operation_handler.doc_options_end(help_command=help_command)
-        rendered = help_command.doc.getvalue().decode('utf-8')
-        self.assertIn(
-            "See :doc:`'aws help' </reference/index>` for descriptions of "
-            "global parameters", rendered
-        )
+        operation_handler.doc_global_option(help_command=help_command)
+        rendered = help_command.doc.getvalue()
+        rendered = rendered.decode('utf-8')
+        for arg in help_command.arg_table:
+            arg_name = help_command.arg_table.get(arg).cli_name
+            type_name = help_command.arg_table.get(arg).cli_type_name
+            target = f'``{arg_name}`` ({type_name})'
+            self.assertIn(target, rendered)
+
+    def test_not_includes_global_options_html(self):
```

Comment:
* I saw the same thing happening elsewhere, but upon a closer look I realize that the tests were separate because the texts were slightly different between `man` and `html`. I've made them into single tests.
* That's a good point about the exact conditions that trigger them. The logic is based on whether or not the help command has an empty `command_table`. Changing the tests to reflect that